plugins/woocommerce/src/Blocks/BlockTypes/CheckoutExpressPaymentBlock.php

```diff
@@ -11,4 +13,54 @@ class CheckoutExpressPaymentBlock extends AbstractInnerBlock {
 	 * @var string
 	 */
 	protected $block_name = 'checkout-express-payment-block';
+
+	/**
+	 * Uniform default_styles for the express payment buttons
+	 *
+	 * @var boolean
+	 */
+	protected $default_styles = null;
+
+	/**
+	 * Initialise the block
+	 */
+	protected function initialize() {
+		parent::initialize();
+
+		$this->default_styles = array(
+			'showButtonStyles'   => false,
+			'buttonHeight'       => '48',
+			'buttonBorderRadius' => '4',
+		);
+
+		add_option(
+			'woocommerce_express_payment_settings',
+			$this->default_styles
+		);
+
+		add_action( 'save_post', array( $this, 'update_express_payment_settings' ), 10, 2 );
+	}
+
+	/**
+	 * Persist the attributes of the express ppayment block to wp options
+	 *
+	 * @param int     $post_id Post ID.
+	 * @param WP_Post $post Post object.
+	 */
+	public function update_express_payment_settings( $post_id, $post ) {
+		CartCheckoutUtils::update_express_payment_settings( $post_id, $post, 'checkout', $this->default_styles );
```

Comment:
This didn't work for me btw. I edited the settings in Checkout, they didn't auto apply in Cart, it wasn't even enabled, I edited settings in Cart, they only applied to Cart, for frontend and editor.

Reply:
Thanks for spotting this, there was a bug in the sync code. I pushed some changes that should make it work. Please test this again.